### PR TITLE
Adding INITIAL Release react-timeline-3rdparty. SPFx 1.17.0 (conflicts revolve from main)

### PR DESCRIPTION
**Pull request by jtlivio**
# SAMPLE

**react-timeline-3rdparty**

## SPFx 1.17.0

|        Q        |                    A                    |
| --------------- | --------------------------------------- |

| New sample?     | yes                              |

## What's in this Pull Request?

This web part is a sample in how you can create a Beautiful TIMELINE with dynamic values from a SharePoint List
This sample show how it's possible to use other controls provided in the GitHub and implement in the SPFx. 

- [Uses react-vertical-timeline-component](https://github.com/stephane-monnot/react-vertical-timeline) [MIT Licence](https://mit-license.org/)
- [Uses PnPJs V3](https://pnp.github.io/pnpjs/getting-started/)
- In this sample I'm not using Caching from the PnPJs V3, feel free to change

### CODE REVIEW

**Review by hugoabernier — CHANGES_REQUESTED** [review]
Can you please change your PR to only affect the react-timeline-3rdparty folder. It should not touch any other folders.

**Comment by jtlivio** [comment]
@hugoabernier sure, thank you

**Review by hugoabernier — CHANGES_REQUESTED** [review]

File: samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+        description,
+        isDarkTheme,
+        environmentMessage,
+        hasTeamsContext,
+        userDisplayName
+      } = this.props;
+
+      return (
+        <section className={`${styles.myTimeline} ${hasTeamsContext ? styles.teams : ''}`}>
+          <div className={styles.welcome}>
```
[the remark this answers is not in the document]
Comment (jtlivio) [reply samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx:68]: @hugoabernier thank you

File: samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+            <h2>Well done, {escape(userDisplayName)}!</h2>
+            <div>{environmentMessage}</div>
+            <div>Web part property value: <strong>{escape(description)}</strong></div>
+          </div>
+          <div>
+            <br /><br /><br />
+            <div style={BackStyle}>
+              <VerticalTimeline>
+                <VerticalTimelineElement
+                  iconStyle={{ background: 'rgb(16, 204, 82)', color: '#fff' }}
```
[the remark this answers is not in the document]
Comment (jtlivio) [reply samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx:79]: @hugoabernier thank you

File: samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx [hunk 3]
```diff
@@ -0,0 +1,156 @@
...
+                  icon={<StarIcon />}
+                />
+
+                {this.state.items.map((item, idx) => {
+                  return (
+                    <VerticalTimelineElement
+                      key={idx}
+                      className="vertical-timeline-element--work"
```
[the remark this answers is not in the document]
Comment (jtlivio) [reply samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx:87]: @hugoabernier thank you

File: samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+import { VerticalTimeline, VerticalTimelineElement } from 'react-vertical-timeline-component';
+import 'react-vertical-timeline-component/style.min.css';
+import StarIcon from '@material-ui/icons/Star';
+
+import "../../myTimeline/timelineStyle.css";
+import * as CSS from "csstype";
+import { Computer, NoteRounded, SquareFootOutlined } from '@material-ui/icons';
+
+const BackStyle: CSS.Properties<string | number> = {
+  background: 'rgb(227, 227, 227)'
```
[the remark this answers is not in the document]
Comment (jtlivio) [reply samples/react-timeline-3rdparty/src/webparts/myTimeline/components/MyTimeline.tsx:24]: @hugoabernier thank you

**Pull request closed without merging**